src/Models/ShallowWaterModels/one_dimensional_shallow_water_model.jl

```diff
@@ -1,46 +1,43 @@
-# # One dimensional ShallowWater example
-#
-# ## Model setup
+# # One dimensional shallow water example
```

Comment:
Let's take this file out of the source code

Reply:
I tried to do that and it doesn't appear in the branch when I view it on github.  But if you still see it then I did something wrong.

Reply:
Ah maybe its gone after all!